auth-web/devops/vaults.env

```diff
@@ -13,6 +13,7 @@ VUE_APP_ONE_STOP_URL="op://relationship/$APP_ENV/auth-web/ONE_STOP_URL"
 
 #vaults web-url
 VUE_APP_REGISTRY_HOME_URL="op://web-url/$APP_ENV/registry/REGISTRY_HOME_URL"
+VUE_APP_REGISTRY_URL="op://web-url/$APP_ENV/registry/REGISTRY_URL"
```

Comment:
Are lines 15 and 16 the same URL?

Reply:
no, registries home url direct to dev.bcros.ca, registry_url direct to bc registries dashboard